tests/kuttl/common/assert_keystone_deployment.yaml

```diff
@@ -0,0 +1,134 @@
+#
+# Check for:
+#
+# - 1 KeystoneAPI CR
+# - 1 Pod for KeystoneAPI CR
```

Comment:
might also want to check for the created service and route?

Reply:
Done. When checking the keystone service, I saw that there are actually 3: `keystone-admin`, `keystone-internal` and `keystone-public`, and the same is true for routes. I added checks for all three in services and routes. Also, I tried to add all the relevant fields, but I'm not 100% sure that I added all the needed ones, or whether all that I added are really relevant.